### DISCUSSION TITLE
Saving pictures

### DISCUSSION THREAD

**Discussion by GCAT01** [FossifyOrg/General-Discussion#491] [Q&A] [answered]
Before I open a bug report, I just wanted to check if I'm missing how to save received pictures. I'm coming from Textra, which had an option to auto-save all received media. If I received a picture and long press on it, there is no option to save it to my phone. Am I missing something?

**Comment by Aga-C** [comment]
After long pressing on the picture, there is a floppy button on the top bar.

**Reply by GCAT01** [reply]
If I long press on a message with one picture, I can save it if I press the 3 dots (I do not see an icon to save). If the message has multiple pictures, there is no option to save them at all (icon or in 3 dot menu).

**Reply by Aga-C** [reply, marked as answer]
Yes, that's the action I meant. Probably you have a smaller screen, so the app decided to hide it in the menu.

Saving multiple images at once is, in fact, missing. You can raise a feature request for that.
